src/components/carousel/SimpleCarousel.js

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import Slider from "react-slick";
+
+export default function SimpleCarousel(props) {
```

Comment:
this is almost the same code as the Carousel component

you should pass the settings as a props

then you only need one component

Reply:
you are right!
nice comment, done